api/file.go

```diff
@@ -15,6 +15,15 @@ import (
 	"github.com/mattermost/platform/utils"
 )
 
+var UNSAFE_CONTENT_TYPES = [...]string{
+	"application/javascript",
+	"application/ecmascript",
+	"text/javaScript",
```

Comment:
Ugh, I fixed it, copied the one from api4, and then fixed that one, so I missed this one. It's fixed now